apps/salonchromaappt/manifest.yaml

```diff
@@ -0,0 +1,8 @@
+---
+id: salonchromaappt
+name: SalonChromaAppt
+summary: Show Weeks Out App
+desc: Example of how to show appointment dates in future. Shows the 4, 6, & 8 week out dates for new appointments at a Salon.
```

Comment:
If this is meant to be used to demonstrate what a future app could look like, it might be better to use the Tidbyt API to push a custom graphic to the device. More info on that [over here](https://tidbyt.dev/docs/integrate/pushing-apps).

Reply:
So my friend runs a salon and all the stylist have manual boards that show the date 4, 6, and 8 weeks out to allow customers to see so they can schedule their next appointment. The idea here is I hard coded the 4,6, & 8 weeks out calculations based on the current day with a custom image but in v2 give user the option to specify their own image and date frames for the 4, 6, & 8 weeks out. The API will not work here to push as it calculates the 4, 6, & 8 weeks out and needs to be an app.